lib/ddtrace/contrib/rails/configuration/settings.rb

```diff
@@ -64,9 +64,17 @@ class Settings < Contrib::Configuration::Settings
             end
           end
 
+          option :web_sockets_service do |o|
```

Comment:
I think it would be wise to avoid adding this option here. The other ones in this class are really for backwards compatibility only; new configuration options for Rails components should only be on the integration for that Rails component, then configured through block syntax. e.g.

```ruby
c.use :rails do |rails|
  rails.action_cable service_name: 'my-service'
  # OR
  rails.action_cable do |action_cable|
    action_cable.service_name = 'my-service'
  end
end
```

We don't support configuring integrations like this quite yet, so users will have to configure `action_cable` in its own `c.use :action_cable, service_name: 'my-service'` until we do. Doing so will help us avoid a deprecation and breaking change.

Reply:
Using new pattern now.